environment/src/main/java/com/sequenceiq/environment/platformresource/v1/converter/PlatformRegionsToRegionV1ResponseConverter.java

```diff
@@ -54,12 +55,31 @@ public RegionResponse convert(CloudRegions source) {
             }
         }
 
+        Collections.sort(regions);
+        Collections.sort(locations);
+        Collections.sort(k8sSupportedLocations);
+
+        availabilityZones = availabilityZones
+                .entrySet()
+                .stream()
+                .sorted(Comparator.comparing(o -> o.getKey()))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue,
+                        (oldValue, newValue) -> oldValue, LinkedHashMap::new));
```

Comment:
You can make a function for this. As I see these lines are the same as for `dispalyNames`